Pod/Classes/SideMenuManager.swift

```diff
@@ -42,6 +42,12 @@ open class SideMenuManager : NSObject {
      */
     open static var menuPresentMode: MenuPresentMode = .viewSlideOut
     
+    /// Retrieves each already created view controller to front when called. Requires class restoration identifiers. Defaults to false.
+    open static var menuPreserveViewOnPush = false
+    
+    /// Mimics a tab function, by instantly presenting the view controller. Defaults to false.
+    open static var menuTabMode = false
```

Comment:
Understood. Try this fix instead of `menuTabMode`:

Try using the following on UISideMenuNavigationController:143:

```
let areAnimationsEnabled = UIView.areAnimationsEnabled
UIView.setAnimationsEnabled(true)
UIView.animate(withDuration: SideMenuManager.menuAnimationDismissDuration, animations: { () -> Void in
       SideMenuTransition.hideMenuStart()
})
UIView.setAnimationsEnabled(areAnimationsEnabled)
```